lib/unified_health_data/service.rb

```diff
@@ -145,6 +141,23 @@ def fetch_observations(record)
       end
     end
 
+    def fetch_observation_value(obs)
+      type, text = if obs['valueQuantity']
+                     ['quantity', "#{obs['valueQuantity']['value']} #{obs['valueQuantity']['unit']}"]
+                   elsif obs['valueCodeableConcept']
+                     ['codeable-concept', obs['valueCodeableConcept']['text']]
+                   elsif obs['valueString']
+                     ['string', obs['valueString']]
+                   elsif obs['valueDateTime']
+                     ['date-time', obs['valueDateTime']]
+                   elsif obs['valueAttachment']
```

Comment:
@acrollet i _think_ we want to raise and handle an exception here so we can know if/when this type ever comes across the wire?

Reply:
✔️